### ISSUE TITLE
Asus Tinker Board | GPU acceleration drivers

### ISSUE THREAD

**Issue by MichaIng** [MichaIng/DietPi#3094]
#### Is your feature request related to a problem? Please describe:

- Our new Buster image lacks GPU acceleration support.

#### Describe the solution you'd like:

- We must find some drivers.

#### Describe alternatives you've considered:

- Leave as is, however would be a shame to have that GPU power unused.

#### Additional context

- Buster repo might already ship what we need, however testers required: https://github.com/MichaIng/DietPi/issues/2977#issuecomment-518019093
```
G_AGI mali-t76x-fbdev-driver
[[ -f '/etc/X11/xorg.conf' ]] || cat << _EOF_ > /etc/X11/xorg.conf
Section "Device"
	Identifier	"Mali-Fbdev"
	Driver		"fbdev"
	Option		"fbdev"		"/dev/fb0"
EndSection
_EOF_
```
Or
```
G_AGI mali-t76x-x11-driver
```

**Comment by MichaIng** [comment]
Alternative, using the driver packages provided by RockChip: https://github.com/rockchip-linux/rk-rootfs-build/tree/master/packages/armhf/libmali

Even other graphics driver and software packages are provided: https://github.com/rockchip-linux/rk-rootfs-build/tree/master/packages/armhf

**Comment by MichaIng** [comment]
Drivers by RockChip have been implemented with v6.26. Testing still required, especially Xserver integration etc, however will mark this as closed.
